sphinx/builders/latex/__init__.py

```diff
@@ -390,6 +404,27 @@ def copy_image_files(self):
                     logger.warning(__('cannot copy image file %r: %s'),
                                    path.join(self.srcdir, src), err)
 
+    def write_message_catalog(self):
+        # type: () -> None
+        def babel(definition):
```

Comment:
I think `babel()` is not good naming. I think `addtocaptions()` might be better.

Reply:
+1 for `addtocaptions()`. Or `addtomessages()`?